src/@chakra-ui/gatsby-plugin/foundations/colors.ts

```diff
@@ -11,7 +11,7 @@ const colors = {
   },
   blue: {
     100: "#dedeff",
-    300: "#ababfe",
+    300: "#8282ff",
```

Comment:
This color value is part of the current version of the DS

Reply:
FYI this is already updated in `dev`